cmd/aws-vpc-cni/main.go

```diff
@@ -49,15 +50,18 @@ import (
 
 	"github.com/containernetworking/cni/pkg/types"
 
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/cniutils"
 	"github.com/aws/amazon-vpc-cni-k8s/utils"
 	"github.com/aws/amazon-vpc-cni-k8s/utils/cp"
-	"github.com/aws/amazon-vpc-cni-k8s/utils/imds"
 )
 
 const (
 	egressPluginIpamSubnetV4     = "169.254.172.0/22"
+	egressPluginIpamSubnetV6     = "fd00::/8"
```

Comment:
Let's use a smaller ULA range here, say `fd00::/118`

Reply:
I think narrow it down is a good idea, but not sure if fd00::/118 is a good choice.

`"CreateTransitGatewayConnectPeerRequest$InsideCidrBlocks": "The range of inside IP addresses that are used for BGP peering. You must specify a size /29 IPv4 CIDR block from the 169.254.0.0/16 range. The first address from the range must be configured on the appliance as the BGP IP address. You can also optionally specify a size /125 IPv6 CIDR block from the fd00::/8 range.`